arango/collection.py

```diff
@@ -306,6 +306,7 @@ def configure(
         schema: Optional[Json] = None,
         replication_factor: Optional[int] = None,
         write_concern: Optional[int] = None,
+        computed_values: Optional[Jsons] = None,
```

Comment:
small nitpick: we have `computed_values` as the parameter name here, but have `computedValues` for the `create_collection` method in `database.py`:

https://github.com/ArangoDB-Community/python-arango/blob/main/arango/database.py#L977